frontend/pages/organizations/[organizationUrl].js

```diff
@@ -224,14 +235,17 @@ function OrganizationLayout({
             {texts.members_of_organization}
           </Typography>
           {canEdit && (
-            <Button
-              className={classes.editButton}
-              variant="contained"
-              color="primary"
+            <Link
               href={getLocalePrefix(locale) + "/manageOrganizationMembers/" + organization.url_slug}
             >
-              {texts.manage_members}
-            </Button>
+              <EditSharpIcon
+                className={classes.button}
+                variant="contained"
+                color="primary"
+              >
+                {texts.manage_members}
+              </EditSharpIcon>
+            </Link>
```

Comment:
Same as above. Would use a button for normal screens and an IconButton for mobile. As an icon, I'd use the `GroupAddIcon` to not have the same Icon as with edit and because we're also using that to manage members of a group chat.